lib/ironic_tls_setup.sh

```diff
@@ -31,6 +39,9 @@ if [ "${IRONIC_TLS_SETUP}" == "true" ]; then
     if [ ! -f "${IRONIC_INSPECTOR_CACERT_FILE}" ]; then
         openssl req -x509 -new -nodes -key "${IRONIC_INSPECTOR_CAKEY_FILE}" -sha256 -days 1825 -out "${IRONIC_INSPECTOR_CACERT_FILE}" -subj /CN="ironic inspector CA"/
     fi
+    if [ ! -f "${MARIADB_CACERT_FILE}" ]; then
```

Comment:
You're missing the CAKEY generation for mariadb

Reply:
Fixed!. Thanks